index.html

```diff
@@ -62,6 +74,30 @@ <h3>Let's Go For A Walk</h3>
 
     </section>
 
+    <div class="signup-form">
+        <form action="action_page.php">
+      
+          <label for="fname">First Name</label>
+          <input type="text" id="fname" name="firstname" placeholder="Your name..">
+      
+          <label for="lname">Last Name</label>
+          <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+      
+          <label for="country">Country</label>
+          <select id="country" name="country">
```

Comment:
In retrospect, you could have also added one of these for magazine subscriptions 👍